imod/tests/test_mf6/test_multimodel/test_mf6_modelsplitter.py

```diff
@@ -71,3 +72,33 @@ def test_slice_model_unstructured(circle_model):
     for submodel_label in unique_labels:
         active_count = new_models[submodel_label].domain.sel({"layer": 1}).count()
         assert label_counts[submodel_label] == active_count.values
+
+
+@pytest.mark.usefixtures("flow_transport_simulation")
+def test_slice_model_with_auxiliary_variables(tmp_path, flow_transport_simulation):
+    flow_simulation = flow_transport_simulation
+    flow_simulation.pop("tpt_a")
+    flow_simulation.pop("tpt_b")
+    flow_simulation.pop("tpt_c")
+    flow_simulation.pop("tpt_d")
+    flow_simulation.pop("transport_solver")
```

Comment:
Note: You are mutating a fixture which has "session" as scope, meaning this results in unwarranted side-effects for other tests, which happen after this. I think you should alter the scope of the fixture to ``"function"``

Reply:
done